test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -3008,6 +3009,32 @@ public async Task GetStatus_ShowInputFalse(string storageProvider)
             }
         }
 
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task GetStatus_testeventTypes()
+        {
+            using (var host = TestHelpers.GetJobHost(
+                   this.loggerProvider,
+                   testName: "GetStatus_testeventTypes",
+                   enableExtendedSessions: false))
+            {
+                await host.StartAsync();
+                var client = await host.StartOrchestratorAsync(nameof(TestOrchestrations.SayHelloWithActivity), "foo", this.output);
+                await client.WaitForCompletionAsync(this.output);
+                var eventList = new List<EventType> { EventType.TaskScheduled, EventType.TaskCompleted };
+                var status = await client.InnerClient.GetStatusAsync(client.InstanceId, showHistory: true, showHistoryOutput: false, showInput: false, eventList);
+
+                var eventTypeName1 = status.History[0].Value<string>("EventType");
+                var eventType1 = (EventType)Enum.Parse(typeof(EventType), eventTypeName1, true);
+                var eventTypeName2 = status.History[1].Value<string>("EventType");
+                var eventType2 = (EventType)Enum.Parse(typeof(EventType), eventTypeName2, true);
+                Assert.Equal(EventType.TaskScheduled, eventType1);
+                Assert.Equal(EventType.TaskCompleted, eventType2);

```

Comment:
Should we add another Assert statement to check that we only received `TaskScheduled` and `TaskCompleted` events? Maybe check the length of `status.History`?